packages/flutter_tools/lib/src/compile.dart

```diff
@@ -453,10 +481,13 @@ class ResidentCompiler {
         ? _mapFilename(request.mainPath, packageUriMapper) + ' '
         : '';
     _server.stdin.writeln('recompile $mainUri$inputKey');
+    printTrace('<- recompile $mainUri$inputKey');
     for (String fileUri in request.invalidatedFiles) {
       _server.stdin.writeln(_mapFileUri(fileUri, packageUriMapper));
+      printTrace('<- ${_mapFileUri(fileUri, packageUriMapper)}');
     }
     _server.stdin.writeln(inputKey);
+    printTrace('<- $inputKey');
```

Comment:
What are the trace statements for? These only show up in verbose logs

Reply:
@aam

Reply:
They show interaction between frontend server and compiler client.

Reply:
Even if logs are are hidden we're still spending time building the strings, quite a lot in the case of the initial reload

Reply:
Do you know time spent here is noticeable? I found that diagnostic to be useful, which is why I added it.

Reply:
There is absolutely no reason to leave debugging code in production here

Reply:
I thin we frequently use `printTrace` when running in verbose mode we want to get clarity about what is going with the system to help troubleshoot issues for end users. This is no different.